### PR TITLE
Apply "readability-identifier-naming" clang-tidy check

### PR DESCRIPTION
**Pull request by florian-kuebler**
Let's agree on naming: Only constexpr is "kCamelCase", all the other variables should be snake_case. That is a bit annoying for "static const" variables, that are effectively also constants, but clang-tidy does not support that, yet.

This change does intentionally not touch method names, yet. There will be likely a follow-up for that.

This needs to be reviewed a bit more carefully. At some places, I switched to constexpr, when appropriate, or added a const modifier.

Test: Compile

### CODE REVIEW

File: src/MemoryTracing/MemoryTracingUtilsTest.cpp [hunk 1]
```diff
@@ -28,13 +28,13 @@ MATCHER_P(MemoryProtoEq, expected, "") {
...
-  const int kMemTotal = 16396576;
-  const int kMemFree = 11493816;
-  const int kMemAvailable = 14378752;
-  const int kBuffers = 71540;
-  const int kCached = 3042860;
+  const int mem_total = 16396576;
+  const int mem_free = 11493816;
+  const int mem_available = 14378752;
+  const int buffers = 71540;
+  const int cached = 3042860;
```
Comment (beckerhe) [inline src/MemoryTracing/MemoryTracingUtilsTest.cpp:35]: I think you could make these constexpr instead and avoid the renaming.

File: src/MemoryTracing/MemoryTracingUtilsTest.cpp [hunk 2]
```diff
@@ -86,61 +86,61 @@ Hugetlb:               0 kB
...
-    EXPECT_THAT(system_memory_usage, MemoryProtoEq(kExpectedSystemMemoryUsage));
+    EXPECT_THAT(system_memory_usage, MemoryProtoEq(expected_system_memory_usage));
   }
 }
 
 TEST(MemoryUtils, UpdateSystemMemoryUsageFromVmStat) {
-  const int kPageFaults = 123456789;
-  const int kMajorPageFaults = 123456;
+  const int page_faults = 123456789;
+  const int major_page_faults = 123456;
```
Comment (beckerhe) [inline src/MemoryTracing/MemoryTracingUtilsTest.cpp:141]: Same here. constexpr would be better

File: src/MemoryTracing/MemoryTracingUtilsTest.cpp [hunk 3]
```diff
@@ -294,96 +294,98 @@ balloon_migrate 3482
...
-    EXPECT_THAT(system_memory_usage, MemoryProtoEq(kExpectedSystemMemoryUsage));
+    EXPECT_THAT(system_memory_usage, MemoryProtoEq(expected_system_memory_usage));
   }
 }
 
 TEST(MemoryUtils, UpdateProcessMemoryUsageFromProcessStat) {
-  const int kMinorPageFaults = 20;
-  const int kMajorPageFaults = 1;
+  const int minor_page_faults = 20;
+  const int major_page_faults = 1;
```
Comment (beckerhe) [inline src/MemoryTracing/MemoryTracingUtilsTest.cpp:342]: constexpr

File: src/MemoryTracing/MemoryTracingUtilsTest.cpp [hunk 4]
```diff
@@ -294,96 +294,98 @@ balloon_migrate 3482
...
+        UpdateProcessMemoryUsageFromProcessStat(empty_process_stat, &process_memory_usage);
     EXPECT_TRUE(updating_result.has_error());
-    EXPECT_THAT(process_memory_usage, MemoryProtoEq(kExpectedProcessMemoryUsage));
+    EXPECT_THAT(process_memory_usage, MemoryProtoEq(expected_process_memory_usage));
   }
 }
 
 TEST(MemoryUtils, ExtractRssAnonFromProcessStatus) {
-  const int kRssAnonKb = 10264;
+  const int rss_anon_kb = 10264;
```
Comment (beckerhe) [inline src/MemoryTracing/MemoryTracingUtilsTest.cpp:386]: constexpr

File: src/MemoryTracing/MemoryTracingUtilsTest.cpp [hunk 5]
```diff
@@ -478,42 +480,42 @@ TEST(MemoryUtil, GetProcessMemoryCGroupName) {
...
 TEST(MemoryUtils, UpdateCGroupMemoryUsageFromMemoryStat) {
-  const int kRssInBytes = 245760;
-  const int KMappedFileInBytes = 1234;
-  const int kPageFaults = 1425;
-  const int kMajorPageFaults = 1;
-  const int kUnevictableInBytes = 0;
-  const int kInactiveAnonInBytes = 16384;
-  const int kActiveAnonInBytes = 253952;
-  const int kInactiveFileInBytes = 3678;
-  const int kActiveFileInBytes = 12288;
```
Comment (beckerhe) [inline src/MemoryTracing/MemoryTracingUtilsTest.cpp:514]: constexpr

File: src/OrbitGl/CaptureViewElementTest.cpp [hunk 6]
```diff
@@ -99,10 +99,10 @@ class UnitTestCaptureViewContainerElement : public CaptureViewElementMock {
 };
 
 TEST(CaptureViewElementTesterTest, PassesAllTestsOnExistingElement) {
-  const int kChildCount = 2;
+  const int child_count = 2;
```
Comment (beckerhe) [inline src/OrbitGl/CaptureViewElementTest.cpp:102]: constexpr

File: src/OrbitGl/CaptureViewElementTest.cpp [hunk 7]
```diff
@@ -124,8 +124,8 @@ TEST(CaptureViewElement, ContainsPointRecursively) {
 }
 
 TEST(CaptureViewElement, ContainsPointRecursivelyForElementsOutsideOfTheParent) {
-  const int kChildCount = 2;
-  UnitTestCaptureViewContainerElement elem(nullptr, &kViewport, &kLayout, kChildCount);
+  const int child_count = 2;
```
Comment (beckerhe) [inline src/OrbitGl/CaptureViewElementTest.cpp:127]: constexpr

File: src/OrbitGl/CaptureViewElementTest.cpp [hunk 8]
```diff
@@ -136,33 +136,33 @@ TEST(CaptureViewElement, ContainsPointRecursivelyForElementsOutsideOfTheParent)
 }
 
 TEST(CaptureViewElement, IsMouseOver) {
-  const int kChildCount = 2;
-  UnitTestCaptureViewContainerElement container_elem(nullptr, &kViewport, &kLayout, kChildCount);
+  const int child_count = 2;
```
Comment (beckerhe) [inline src/OrbitGl/CaptureViewElementTest.cpp:139]: constexpr

File: src/OrbitGl/CaptureViewElementTest.cpp [hunk 9]
```diff
@@ -191,18 +191,18 @@ TEST(CaptureViewElement, MouseWheelEventRecursesToCorrectChildren) {
   using ::testing::Exactly;
   using ::testing::Return;
 
-  const int kChildCount = 3;
-  UnitTestCaptureViewContainerElement container_elem(nullptr, &kViewport, &kLayout, kChildCount);
+  const int child_count = 3;
```
Comment (beckerhe) [inline src/OrbitGl/CaptureViewElementTest.cpp:194]: constexpr

File: src/OrbitGl/CaptureViewElementTest.cpp [hunk 10]
```diff
@@ -191,18 +191,18 @@ TEST(CaptureViewElement, MouseWheelEventRecursesToCorrectChildren) {
...
-  const Vec2 kPosOnChild1(10, kLeafElementHeight + kMarginAfterChild);
-  const Vec2 kPosOnChild2(10, (kLeafElementHeight + kMarginAfterChild) * 2);
+  const Vec2 pos_outside(-1, -1);
+  const Vec2 pos_between_children(10, kLeafElementHeight + 1);
+  const Vec2 pos_on_child0(10, kLeafElementHeight - 1);
+  const Vec2 pos_on_child1(10, kLeafElementHeight + kMarginAfterChild);
+  const Vec2 pos_on_child2(10, (kLeafElementHeight + kMarginAfterChild) * 2);
 
-  const int kDelta = 1;
+  const int delta = 1;
```
Comment (beckerhe) [inline src/OrbitGl/CaptureViewElementTest.cpp:205]: constexpr

File: src/OrbitGl/CaptureViewElementTester.cpp [hunk 11]
```diff
@@ -59,25 +59,25 @@ void orbit_gl::CaptureViewElementTester::SimulateDrawLoopAndCheckFlags(CaptureVi
 
 void orbit_gl::CaptureViewElementTester::TestWidthPropagationToChildren(
     CaptureViewElement* element) {
-  const float kWidth = 100, kUpdatedWidth = 50;
+  const float width = 100, updated_width = 50;
```
Comment (beckerhe) [inline src/OrbitGl/CaptureViewElementTester.cpp:62]: constexpr

File: src/OrbitGl/CaptureWindow.cpp [hunk 12]
```diff
@@ -114,15 +114,15 @@ void CaptureWindow::PreRender() {
     // the root element (time graph) of the tree.
     // During loading or capturing, only a single layouting loop is executed as we're
     // streaming in data from a seperate thread (for performance reasons)
-    const int kMaxLayoutLoops =
+    const int max_layout_loops =
```
Comment (beckerhe) [inline src/OrbitGl/CaptureWindow.cpp:117]: constexpr

File: src/OrbitGl/CaptureWindow.cpp [hunk 13]
```diff
@@ -344,9 +344,9 @@ void CaptureWindow::MouseWheelMovedHorizontally(int x, int y, int delta, bool ct
 
 void CaptureWindow::KeyPressed(unsigned int key_code, bool ctrl, bool shift, bool alt) {
   GlCanvas::KeyPressed(key_code, ctrl, shift, alt);
-  const float kPanRatioPerLeftAndRightArrowKeys = 0.1f;
-  const float kScrollingRatioPerUpAndDownArrowKeys = 0.05f;
-  const float kScrollingRatioPerPageUpAndDown = 0.9f;
+  const float pan_ratio_per_left_and_right_arrow_keys = 0.1f;
+  const float scrolling_ratio_per_up_and_down_arrow_keys = 0.05f;
+  const float scrolling_ratio_per_page_up_and_down = 0.9f;
```
Comment (beckerhe) [inline src/OrbitGl/CaptureWindow.cpp:349]: constexpr

File: src/OrbitGl/CaptureWindow.cpp [hunk 14]
```diff
@@ -681,11 +681,11 @@ void CaptureWindow::RenderHelpUi() {
       {time_graph_layout_->GetFontSize(), Color(255, 255, 255, 255), -1.f /*max_size*/},
       &text_bounding_box_pos, &text_bounding_box_size);
 
-  const Color kBoxColor(50, 50, 50, 243);
-  const float kMargin = 15.f;
-  const float kRoundingRadius = 20.f;
+  const Color box_color(50, 50, 50, 243);
+  const float margin = 15.f;
+  const float rounding_radius = 20.f;
```
Comment (beckerhe) [inline src/OrbitGl/CaptureWindow.cpp:686]: constexpr

File: src/OrbitGl/GlSlider.cpp [hunk 15]
```diff
@@ -86,17 +86,17 @@ void GlSlider::SetNormalizedLength(float length_ratio)  // [0,1]
 }
 
 Color GlSlider::GetLighterColor(const Color& color) {
-  const float kLocalGradientFactor = 1.0f + kGradientFactor;
-  return {static_cast<unsigned char>(color[0] * kLocalGradientFactor),
-          static_cast<unsigned char>(color[1] * kLocalGradientFactor),
-          static_cast<unsigned char>(color[2] * kLocalGradientFactor), 255};
+  const float local_gradient_factor = 1.0f + kGradientFactor;
```
Comment (beckerhe) [inline src/OrbitGl/GlSlider.cpp:89]: constexpr

File: src/OrbitGl/GlSlider.cpp [hunk 16]
```diff
@@ -86,17 +86,17 @@ void GlSlider::SetNormalizedLength(float length_ratio)  // [0,1]
...
+          static_cast<unsigned char>(color[1] * local_gradient_factor),
+          static_cast<unsigned char>(color[2] * local_gradient_factor), 255};
 }
 
 Color GlSlider::GetDarkerColor(const Color& color) {
-  const float kLocalGradientFactor = 1.0f - kGradientFactor;
-  return {static_cast<unsigned char>(color[0] * kLocalGradientFactor),
-          static_cast<unsigned char>(color[1] * kLocalGradientFactor),
-          static_cast<unsigned char>(color[2] * kLocalGradientFactor), 255};
+  const float local_gradient_factor = 1.0f - kGradientFactor;
```
Comment (beckerhe) [inline src/OrbitGl/GlSlider.cpp:96]: constexpr

File: src/OrbitGl/GlSlider.cpp [hunk 17]
```diff
@@ -166,10 +166,10 @@ void GlSlider::OnPick(int x, int y) {
 void GlSlider::DrawBackground(PrimitiveAssembler& primitive_assembler, float x, float y,
                               float width, float height) {
   const Color dark_border_color = GetDarkerColor(bar_color_);
-  const float kEpsilon = 0.0001f;
+  const float epsilon = 0.0001f;
```
Comment (beckerhe) [inline src/OrbitGl/GlSlider.cpp:169]: constexpr

File: src/OrbitGl/GlSlider.cpp [hunk 18]
```diff
@@ -183,29 +183,29 @@ void GlSlider::DrawSlider(PrimitiveAssembler& primitive_assembler, float x, floa
   Color color = (IsMouseOver() && PosIsInSlider(mouse_pos_cur_)) ? selected_color_ : slider_color_;
   const Color dark_border_color = GetDarkerColor(bar_color_);
   const Color light_border_color = GetLighterColor(color);
-  const float kEpsilon = 0.0001f;
+  const float epsilon = 0.0001f;
```
Comment (beckerhe) [inline src/OrbitGl/GlSlider.cpp:186]: constexpr

File: src/OrbitGl/GlSlider.cpp [hunk 19]
```diff
@@ -183,29 +183,29 @@ void GlSlider::DrawSlider(PrimitiveAssembler& primitive_assembler, float x, floa
...
 
   Quad light_border_box = MakeBox(Vec2(x + 1.f, y + 1.f), Vec2(width - 2.f, height - 2.f));
 
-  primitive_assembler.AddBox(light_border_box, GlCanvas::kZValueButton - kEpsilon,
+  primitive_assembler.AddBox(light_border_box, GlCanvas::kZValueButton - epsilon,
                              light_border_color, shared_from_this());
 
   // Slider itself
-  const float kSliderOffset = 2.f;
+  const float slider_offset = 2.f;
```
Comment (beckerhe) [inline src/OrbitGl/GlSlider.cpp:199]: constexpr

File: src/OrbitGl/GlSlider.cpp [hunk 20]
```diff
@@ -299,49 +299,49 @@ void GlHorizontalSlider::DoDraw(PrimitiveAssembler& primitive_assembler,
   ShadingDirection shading_direction = ShadingDirection::kTopToBottom;
   DrawSlider(primitive_assembler, start, 0, slider_width, GetSliderWidth(), shading_direction);
 
-  const float kEpsilon = 0.0001f;
+  const float epsilon = 0.0001f;
```
Comment (beckerhe) [inline src/OrbitGl/GlSlider.cpp:302]: constexpr

File: src/OrbitGl/GlSlider.cpp [hunk 21]
```diff
@@ -299,49 +299,49 @@ void GlHorizontalSlider::DoDraw(PrimitiveAssembler& primitive_assembler,
...
 
   // Left / right resize arrows and separator
-  const float kHeightFactor = 2.f;
+  const float height_factor = 2.f;
```
Comment (beckerhe) [inline src/OrbitGl/GlSlider.cpp:305]: constexpr

File: src/OrbitGl/PrimitiveAssembler.cpp [hunk 22]
```diff
@@ -302,8 +302,8 @@ void PrimitiveAssembler::AddQuadBorder(const Quad& quad, float z, const Color& c
 
 void PrimitiveAssembler::GetBoxGradientColors(const Color& color, std::array<Color, 4>* colors,
                                               ShadingDirection shading_direction) {
-  const float kGradientCoeff = 0.94f;
-  Vec3 dark = Vec3(color[0], color[1], color[2]) * kGradientCoeff;
+  const float gradient_coeff = 0.94f;
```
Comment (beckerhe) [inline src/OrbitGl/PrimitiveAssembler.cpp:305]: constexpr

File: src/UserSpaceInstrumentation/InstrumentProcess.cpp [hunk 23]
```diff
@@ -211,7 +211,7 @@ ErrorMessageOr<std::vector<pid_t>> GetNewOrbitThreads(
   // happened in the real target processes.
   // We choose a three second (300 x 10 ms) timeout and query the existing threads every 10 ms.
   constexpr int kNumberOfRetries = 300;
-  std::chrono::milliseconds kWaitingPeriod{10};
+  std::chrono::milliseconds waiting_period{10};
```
Comment (beckerhe) [inline src/UserSpaceInstrumentation/InstrumentProcess.cpp:214]: constexpr

**Review by beckerhe — APPROVED** [review]
Otherwise LGTM

File: src/MemoryTracing/MemoryTracingUtilsTest.cpp (hunk 1 above)
Comment (florian-kuebler) [reply src/MemoryTracing/MemoryTracingUtilsTest.cpp]: Done.

File: src/MemoryTracing/MemoryTracingUtilsTest.cpp (hunk 2 above)
Comment (florian-kuebler) [reply src/MemoryTracing/MemoryTracingUtilsTest.cpp]: Done.

File: src/MemoryTracing/MemoryTracingUtilsTest.cpp (hunk 3 above)
Comment (florian-kuebler) [reply src/MemoryTracing/MemoryTracingUtilsTest.cpp]: Done.

File: src/MemoryTracing/MemoryTracingUtilsTest.cpp (hunk 4 above)
Comment (florian-kuebler) [reply src/MemoryTracing/MemoryTracingUtilsTest.cpp]: Done.

File: src/MemoryTracing/MemoryTracingUtilsTest.cpp (hunk 5 above)
Comment (florian-kuebler) [reply src/MemoryTracing/MemoryTracingUtilsTest.cpp:514]: Done.

File: src/OrbitGl/CaptureViewElementTest.cpp (hunk 6 above)
Comment (florian-kuebler) [reply src/OrbitGl/CaptureViewElementTest.cpp]: Done.

File: src/OrbitGl/CaptureViewElementTest.cpp (hunk 7 above)
Comment (florian-kuebler) [reply src/OrbitGl/CaptureViewElementTest.cpp]: Done.

File: src/OrbitGl/CaptureViewElementTest.cpp (hunk 8 above)
Comment (florian-kuebler) [reply src/OrbitGl/CaptureViewElementTest.cpp]: Done.

File: src/OrbitGl/CaptureViewElementTest.cpp (hunk 9 above)
Comment (florian-kuebler) [reply src/OrbitGl/CaptureViewElementTest.cpp]: Done.

File: src/OrbitGl/CaptureViewElementTest.cpp (hunk 10 above)
Comment (florian-kuebler) [reply src/OrbitGl/CaptureViewElementTest.cpp]: Done.

File: src/OrbitGl/CaptureViewElementTester.cpp (hunk 11 above)
Comment (florian-kuebler) [reply src/OrbitGl/CaptureViewElementTester.cpp]: Done.

File: src/OrbitGl/CaptureWindow.cpp (hunk 12 above)
Comment (florian-kuebler) [reply src/OrbitGl/CaptureWindow.cpp:120]: Thats actually not a constant :)

File: src/OrbitGl/CaptureWindow.cpp (hunk 13 above)
Comment (florian-kuebler) [reply src/OrbitGl/CaptureWindow.cpp]: Done.

File: src/OrbitGl/CaptureWindow.cpp (hunk 14 above)
Comment (florian-kuebler) [reply src/OrbitGl/CaptureWindow.cpp]: Done.

File: src/OrbitGl/GlSlider.cpp (hunk 15 above)
Comment (florian-kuebler) [reply src/OrbitGl/GlSlider.cpp]: Done.

File: src/OrbitGl/GlSlider.cpp (hunk 16 above)
Comment (florian-kuebler) [reply src/OrbitGl/GlSlider.cpp]: Done.

File: src/OrbitGl/GlSlider.cpp (hunk 17 above)
Comment (florian-kuebler) [reply src/OrbitGl/GlSlider.cpp]: Done.

File: src/OrbitGl/GlSlider.cpp (hunk 18 above)
Comment (florian-kuebler) [reply src/OrbitGl/GlSlider.cpp]: Done.

File: src/OrbitGl/GlSlider.cpp (hunk 19 above)
Comment (florian-kuebler) [reply src/OrbitGl/GlSlider.cpp]: Done.

File: src/OrbitGl/GlSlider.cpp (hunk 20 above)
Comment (florian-kuebler) [reply src/OrbitGl/GlSlider.cpp]: Done.

File: src/OrbitGl/GlSlider.cpp (hunk 21 above)
Comment (florian-kuebler) [reply src/OrbitGl/GlSlider.cpp]: Done.

File: src/OrbitGl/PrimitiveAssembler.cpp (hunk 22 above)
Comment (florian-kuebler) [reply src/OrbitGl/PrimitiveAssembler.cpp]: Done.

File: src/UserSpaceInstrumentation/InstrumentProcess.cpp (hunk 23 above)
Comment (florian-kuebler) [reply src/UserSpaceInstrumentation/InstrumentProcess.cpp]: Done.

**Review by florian-kuebler — COMMENTED** [review]
Thanks

**Pull request merged**